src/layer/vulkan/padding_vulkan.cpp

```diff
@@ -349,16 +462,25 @@ int Padding_vulkan::forward(const std::vector<VkImageMat>& bottom_blobs, std::ve
     int _bottom;
     int _left;
     int _right;
+    int _front;
+    int _behind;
     {
         const int* param_data = reference_blob.mapped();
 
         _top = param_data[0];
         _bottom = param_data[1];
         _left = param_data[2];
         _right = param_data[3];
+        if (reference_blob.total() >= 9){
+            _front = param_data[7];
+            _behind = param_data[8];
```

Comment:
the reference_blob contains the padding border constants only.
use param_data[4] and param_data[5] for front and behind param